decide-app/src/main/java/com/decide/app/LaunchInterceptorConditions.java

```diff
@@ -52,8 +52,26 @@ public boolean getLaunchInterceptorCondition0() {
         return false;
     }
 
+    /**
+     * Launch Interceptor Condition 1:
+     * There exists at least one set of three consecutive data points that cannot all be contained
+     * within or on a circle of radius RADIUS1.
+     * @return True if the condition is met, false otherwise.
+     */
     public boolean getLaunchInterceptorCondition1() {
-        return true;
+        if(PARAMETERS.RADIUS1 < 0) throw new IllegalArgumentException();
+        double d1;
+        double d2;
+        double d3;
+        if(NUMPOINTS != POINTS.length) return false;
+        for(int i = 0; i < NUMPOINTS-2; i++){
+            d1 = distance(POINTS[i], POINTS[i+1]);
+            d2 = distance(POINTS[i], POINTS[i+2]);
+            d3 = distance(POINTS[i+1], POINTS[i+2]);
+
+            if(d1 > PARAMETERS.RADIUS1*2 || d2 > PARAMETERS.RADIUS1*2 || d3 > PARAMETERS.RADIUS1*2) return true;

```

Comment:
This is insufficient. Imagine an equilateral triangle with sides 2*RADIUS1, it cannot be contained within a circle of RADIUS1, but the method will return false.
![bild](https://github.com/alexarne/DECIDE/assets/75060613/8cfddbc3-1c9a-4a00-a1aa-ae8dcd17fd55)